pkgs/applications/networking/sync/desync/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchgit }:
+
+buildGoPackage rec {
+  name = "desync-${version}";
+  version = "0.3.0";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/folbricht/desync";
+
+  src = fetchgit {
```

Comment:
Please use `fetchFromGitHub` instead, see other examples in nixpkgs. It is our preferred method to fetch sources from Github.

Reply:
Implemented per request in 4bd41cce3fede23eb3e722821580cc9eb82e7727